controllers/commentsController.js

```diff
@@ -33,18 +31,80 @@ const getByCity = async (req, res) => {
         },
     });
     if (isEmpty(cities)) {
-        res.status(500)
-        return res;
+        return res.status(500);
+    }
+    return res.status(200).json({
+        main: {
+            cityName,
+            comments: cities,
```

Comment:
rename variable from cities to comments, it's confusing
from db you got cities but returned comments